.gitignore

```diff
@@ -32,4 +32,8 @@ migration_backup
 
 # Other
 logs/
-.vscode/
\ No newline at end of file
+.vscode/
+
+db.sqlite3
+dev.py
+base_settings.py
```

Comment:
I think I understand why you put dev.py in the .gitignore cause you are using sqlite instead of postgres, but could you find a way to get your local dev machine working without modifying these settings files?

I think what you need to do is create your own settings file called something like custom_settings.py and put your specific dev settings in it.

Then, go into your manage.py and modify it to:
```
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_project.settings.custom_settings")
```

```py
def main():
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_project.settings.dev")
    try:
        from django.core.management import execute_from_command_line
    except ImportError as exc:
        raise ImportError(
            "Couldn't import Django. Are you sure it's installed and "
            "available on your PYTHONPATH environment variable? Did you "
            "forget to activate a virtual environment?"
        ) from exc
    execute_from_command_line(sys.argv)
```
This way you can have your own settings without issue.

Reply:
This worked. I was able to create a custom settings file

Reply:
I have also removed dev.py, and base_settings.py from gitignore